src/OrchardCore.Modules/OrchardCore.OpenId/Views/OpenIdSettings.Edit.cshtml

```diff
@@ -13,6 +13,16 @@
         <span class="hint">@T["Disables https requirement and uses an ephemeral signing key."]</span>
     </fieldset>
 
+    <fieldset class="form-group" asp-validation-class-for="UseRollingTokens">
+        <div class="form-check">
+            <label class="form-check-label">
+                <input asp-for="UseRollingTokens" class="form-check-input" checked="@Model.UseRollingTokens" />
+                @T["Use Rolling Tokens"]
+            </label>
+        </div>
+        <span class="hint">@T["When this option is enabled, a new refresh token is issued for each refresh token request (and the previous one is automatically revoked)"]</span>
```

Comment:
Should we add a message indicating this option should be enabled by people who know what they are doing?

Reply:
What kind of message?